roles/confluent.test/upgrade_testing/upgrade.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
wondering if we should run pint merge first, and then not include this file in this pr. wdyt?

Reply:
Good idea!